docs/usage.md

```diff
@@ -31,22 +31,21 @@ run_dir
 ...would be represented in the following samplesheet (shown as .tsv for readability)
 
 ```csv title="samplesheet.csv"
-sample  lane  group   fastq_1                                       fastq_2 rundir
-sample1 1     group1  path/to/run_dir/sample1_lane1_group1_r1.fq.gz         path/to/run_dir
-sample2 1     group1  path/to/run_dir/sample2_lane1_group1_r1.fq.gz         path/to/run_dir
-sample3 2     group2  path/to/run_dir/sample3_lane2_group2_r1.fq.gz         path/to/run_dir
-sample4 2     group3  path/to/run_dir/sample4_lane2_group3_r1.fq.gz         path/to/run_dir
+sample  fastq_1                                       fastq_2 rundir          tags
```

Comment:
This example CSV lacks the separating commas, no?

Reply:
It is mentioned above that this is shown as `.tsv` for readability